### PR TITLE
Add remaining resources sections

### PR DESCRIPTION
**Pull request by samderanova**
Added the frontend and starter pack resources sections and removed the placeholder config files.

### CODE REVIEW

File: apps/site/src/views/Resources/Resources.tsx [hunk 1]
```diff
@@ -2,12 +2,17 @@ import styles from "./Resources.module.scss";
...
+import StarterPacks from "./sections/StarterPacks/StarterPacks";
+
 export default function Resources() {
 	return (
 		<div className={styles.resources}>
 			<Landing />
 			<ApiResources />
 			<BackendResources />
+			<FrontendResources />
+			<StarterPacks />
```
Comment (taesungh) [inline apps/site/src/views/Resources/Resources.tsx:15]: Seems like we're repeating nearly identical components. Is there value in keeping them distinct or should we aim to parameterize and abstract?

Comment (samderanova) [reply apps/site/src/views/Resources/Resources.tsx:15]: We could abstract, but I was thinking we release first and then abstract later.

Comment (tyleryy) [reply apps/site/src/views/Resources/Resources.tsx:15]: Agreed, we need fast release.

**Review by tyleryy — APPROVED** [review]
No apparent blockers lgtm.

**Pull request merged**